rest-api/src/main/resources/paths/studies/v5_studies_studyId_enrollments_userId.yml

```diff
@@ -1,3 +1,30 @@
+post:
```

Comment:
Add description indicating that study designers and developers can only edit test accounts.